appveyor.yml

```diff
@@ -17,24 +17,7 @@ init:
 
 before_build:
 - c:\Windows\System32\sc.exe config NetTcpPortSharing start= demand
-
-on_finish:
-- c:\Windows\System32\sc.exe config NetTcpPortSharing start= disabled
-- ps: >-
-    $wc = New-Object System.Net.WebClient
-
-    $testBinPath = "bin\Release\net45"
-
-    $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "src\Castle.Windsor.Tests\$testBinPath\TestResult_Windsor.xml"))
-
-    $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "src\Castle.Windsor.Tests\$testBinPath\TestResult_Windsor_NetCore.xml"))
-
-    $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "src\Castle.Facilities.WcfIntegration.Tests\$testBinPath\TestResult_WcfIntegration.xml"))
-
-    Push-AppveyorArtifact $env:APPVEYOR_BUILD_FOLDER\src\Castle.Windsor.Tests\$testBinPath\TestResult_Windsor.xml
-
-    Push-AppveyorArtifact $env:APPVEYOR_BUILD_FOLDER\src\Castle.Facilities.WcfIntegration.Tests\$testBinPath\TestResult_WcfIntegration.xml
-
```

Comment:
It was also causing pathing issues that could not be picked up locally. Bad reason for a build to fail :'(

Reply:
Agreed that NUnit's console output is much better for failures now so we don't need the last 2, however the "/api/testresults/nunit/..." ones are for the "Tests" tab in AppVeyor. Castle Core only has the first ones.

Reply:
Does anyone look at those test tabs? I dont. Also noticed test output from netcoreapp1.0 would wipe out test results from net45 because of the namespace collisions. Let me know if you use this and I will put it back.

Reply:
Good point, the test tab is pretty pointless because I don't think you can ever navigate to anything including failures unlikely TeamCity.

It doesn't look like one test run wipes out the other, there are duplicate test names under different file names, `Castle.Windsor.Tests.dll` (.NET 4.5) and `Castle.Windsor.Tests.exe` (.NET Core).

But yer I agree let's just get rid of them. Should we also run NUnit's runner with `--noresult` since the command line output is plenty good?

Reply:
https://github.com/castleproject/Windsor/pull/376/commits/336fdc8a2eab972aaccbd18fc11d11c88302b106